template/styles/css/main.css

```diff
@@ -832,15 +832,14 @@ a.disable:hover {
   min-height: 100%; }
 
 .body-content {
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
   position: relative;
-  overflow: hidden;
```

Comment:
Is there a reason overflow: hidden was removed?

Reply:
This is the compiled sass from